docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,810 @@
+-  `Owners` - The addresses that are allowed to sign and submit proposals. These can be can either be EOAs or other smart contract accounts.
+- `Threshold` - The amount of signatures required to submit a proposal.
+
+### Add an Owner
+To add an owner we must first generate the data required to submit a proposal.
+
+Navigate to the [add_owner.json](https://github.com/5afe/safe-tasks/blob/master/examples/add_owner.json) file in the examples directory and an address that you control to `params`.
+
+```json
+[
+    {
+        "to": "0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114",
+        "value": "0",
+        "method": "addOwnerWithThreshold(address,uint256)",
+        "params": [
+            "0x82DdaF3f1fcd3c18F5664cD7fb12bD8C38D5d4ba",
+            "2"
+        ],
+        "operation": 0
+    }
+]
+```
+Next, we will call the `propose-multi` task to create a transaction based on the sample tx input json that adds an owner to the safe.
+
+```zsh
+yarn safe propose-multi --network subnet 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 examples/add_owner.json --export example/addOwner.json
+```
+This will create a new file, `addOwner.json`, in the examples directory.
+
+```json
+{
+  "version": "1.0",
+  "chainId": "99999",
+  "createdAt": 1658276969641,
+  "meta": {
+    "name": "Custom Transactions"
+  },
+  "transactions": [
+    {
+      "to": "0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114",
+      "value": "0",
+      "data": "0x0d582f1300000000000000000000000082ddaf3f1fcd3c18f5664cd7fb12bd8c38d5d4ba0000000000000000000000000000000000000000000000000000000000000002",
+      "operation": 0
+    }
+  ]
+}
+```
+
+Notice the `data` value has the parameters encoded as a single hexadecimal string.
+- `addOwnerWithThreshold` has the function signature `0d582f13`
+- `address` appears in the data as `82ddaf3f1fcd3c18f5664cd7fb12bd8c38d5d4ba`
+- `threshold` appears at the end of the data as `2`
+
+Now we can use the `--data` flag and pass in the `data` above as an argument for our proposal.
+```zsh
+yarn safe propose --network subnet 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 --to 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 --data 0x0d582f1300000000000000000000000082ddaf3f1fcd3c18f5664cd7fb12bd8c38d5d4ba0000000000000000000000000000000000000000000000000000000000000002
+```
+
+Output:
+```zsh
+Running on subnet
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114
+Safe transaction hash: 0x2837eb329c41078c97e2450eabf0b73caae94d08db06a5d9fe2084d33ef3f4cc
+```
+
+As you can see, creating a proposal generates a `Safe transaction hash` which we will use to complete this tutorial.
+
+Next, we will sign and submit our Proposal's tx hash, `0x2837eb329c41078c97e2450eabf0b73caae94d08db06a5d9fe2084d33ef3f4cc`, with the tasks, `sign-proposal` and `submit-proposal`.
```

Comment:
Please check all places for weird upper cases such as this `Proposal's` and correct them.